samples/dependency-injection/aspnetcore/Core_7.2/SampleAutofac/Startup.cs

```diff
@@ -28,26 +28,5 @@ public void Configure(IApplicationBuilder applicationBuilder, IWebHostEnvironmen
         {
             applicationBuilder.UseDeveloperExceptionPage();
         }
-
-        #region RequestHandling
-
-        applicationBuilder.Run(
-            handler: context =>
-            {
-                if (context.Request.Path != "/")
-                {
-                    // only handle requests at the root
-                    return Task.CompletedTask;
-                }
-                var applicationServices = applicationBuilder.ApplicationServices;
-                var endpointInstance = applicationServices.GetService<IMessageSession>();
-                var myMessage = new MyMessage();
-
-                return Task.WhenAll(
-                    endpointInstance.SendLocal(myMessage),
-                    context.Response.WriteAsync("Message sent"));
-            });
-
-        #endregion
```

Comment:
Sorry, I should have looked closer the first time round. I think this is required for the sample to work.

"The sample" in this case being the `SampleAutofac.Core` project.

The solution has 2 projects, one `Sample.Core` and the other `SampleAutofac.Core`. Removing this piece of code here prevents the `SampleAutofac.Core` project from behaving correctly. 

I suggest instead renaming this snippet from `RequestHandling` to `RequestHandlingAutofac` (Apparently I can't make a suggestion on deleted lines in Github??). Making this change will mean that only one snippet will correspond to the `RequestHandling` key.

Reply:
Since we're not using <code>RequestHandlingAutofac</code> as a snippet anywhere, do we still need to enclose that in a region for some reason? If not I think it's a better idea to just have the code in there in order for the sample to behave correctly without any region.

Reply:
Yeah, I think we can drop the region. The Sample.Core snippet is explained just above the autofac code - there doesn't seem to be a compelling reason to keep the region.